src/components/pages/test-pages/floating-card-page/FloatingCardPage.styles.ts

```diff
@@ -0,0 +1,8 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const wrapper: SxProps<Theme> = {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  height: '100vh',
+};
```

Comment:
You can remove testPage, we can test your component in `/teachers` page

Reply:
removed